std/php/_std/Array.hx

```diff
@@ -273,3 +278,8 @@ private extern interface ArrayAccess<K, V> {
 	private function offsetSet(offset:K, value:V):Void;
 	private function offsetUnset(offset:K):Void;
 }
+
+@:native('ArrayIterator')
+private extern class NativeArrayIterator<T> implements Traversable {
```

Comment:
Why private to `Array` again? :)
This is a common PHP native interface which could be used anywhere, not only in `Array`

Reply:
Well, it extends `SeekableIterator`, `Countable` and `Serializable`. Should I add those to php package too to implement it properly?

Reply:
That would be great. Though, if you don't have time to, I can do it myself.

Reply:
I'll add them. Wasn't sure if we wanted all of those in there :)

Reply:
We want externs for all of the PHP native types and functions eventually.